lib/socket.js

```diff
@@ -151,7 +151,9 @@ module.exports = class UDXSocket extends events.EventEmitter {
         host = '::'
         family = 6
         this._port = binding.udx_napi_socket_bind(this._handle, port, host, family, flags)
-      } catch {
+      } catch (error) {
+        if (error.code !== 'EAFNOSUPPORT') throw error
```

Comment:
Whats this error code?

Reply:
To avoid the second bind to throw unnecessarily, but will remove it reconsidering the previous feedback

Reply:
Just realizing without that error check this doesn't fixes the error anymore, now it will always throw `EINVAL` again

Reply:
`EAFNOSUPPORT` means no support for IPv6, if that's the case then it was trying to bind to IPv4 otherwise throw any other error that happened like `EADDRINUSE`

Reply:
The second test, in Linux it says EINVAL and Windows says EFAULT

```
Error: bad address in system call argument
    at UDXSocket.bind (C:\Users\Lucas\Desktop\udx-native\lib\socket.js:159:30)
    at C:\Users\Lucas\Desktop\udx-native\test\socket.js:620:7
    at Test._run (C:\Users\Lucas\Desktop\udx-native\node_modules\brittle\index.js:576:13) {
  code: 'EFAULT'
} 
```

When checking against `EAFNOSUPPORT` is like saying `if (error.code === 'EADDRINUSE') throw error` so it throws early to be more stable